torch/testing/_internal/common_fsdp.py

```diff
@@ -432,6 +440,11 @@ def _train_for_several_steps(
             ), "loss data type should be float32, as the original \
                  parameter data type is float32."
             model.module.run_backward(loss)
+            if norm_type is not None:
+                if isinstance(model, FullyShardedDataParallel):
+                    model.clip_grad_norm_(clip_norm, norm_type)
+                else:
+                    torch.nn.utils.clip_grad_norm_(model.parameters(), clip_norm, norm_type)
```

Comment:
is this to test that DDP and FSDP have parity when clipping gradients? If so can you add some simple logic ensuring gradients were actually clipped (even something as simple as making sure total norm is under clip_norm value)

Reply:
sure.